guides/common/attributes-base.adoc

```diff
@@ -59,8 +59,8 @@
 :customrpmtitle: RPM
 :customssl: SSL
 :customssltitle: SSL
-:Debian: Debian/Ubuntu
-:Debian-context: Debian-Ubuntu
+:DL: Debian/Ubuntu
```

Comment:
I think DL is ambiguous and haven't seen that used elsewhere. Would `deb-os` be more descriptive?

Reply:
I'd like to keep it similar to "EL" here. It's not yet used in upstream but in downstream docs. I've added it as "Debian Linux" and the colleague who reviewed it was like "ah, Debian-like as attribute for Debian and Ubuntu".

Reply:
For me, both `DL` and `deb-os` is fine. There is a parallel between `DL` and `EL`, so no issues from my side.

Reply:
DL is fine!